scala_proto/private/scalapb_aspect.bzl

```diff
@@ -62,6 +62,10 @@ def _compile_scala(
         label.name + "_scalac.statsfile",
         sibling = scalapb_jar,
     )
+    diagnosticsfile = ctx.actions.declare_file(
```

Comment:
maybe add it to the outputs of scrooge?

Reply:
My question from above also applies here